src/components/VIconButton/VIconButton.vue

```diff
@@ -49,6 +49,14 @@ export default {
      * documentation on `VIcon`.
      */
     iconProps: {},
+    /**
++     * `variant` prop to pass to `VButton`. See `VButton` for documentation
++     * and possible values.
++     */
++    variant: {
++      type: String,
++      default: 'plain',
++    },
```

Comment:
There are some `+` signs interfering with the compilation of the component.
```suggestion
    /**
     * `variant` prop to pass to `VButton`. See `VButton` for documentation
     * and possible values.
     */
    variant: {
      type: String,
      default: 'plain',
    },
```